### PR TITLE
created wishlist and their corresponding views

### PR DESCRIPTION
**Pull request by rishabhharod**

### CODE REVIEW

File: app/controllers/carts_controller.rb [hunk 1]
```diff
@@ -1,11 +1,10 @@
 class CartsController < ApplicationController
-  before_action :set_cart,only: [:show]
-
-  def show
+  before_action :set_cart,only: [:index] 
+  def index
```
Comment (vinayaklovanshi) [inline app/controllers/carts_controller.rb:4]: Add line after callbacks

File: app/controllers/wishlist_items_controller.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
+class WishlistItemsController < ApplicationController
+  before_action :set_wishlist
+  def create
```
Comment (vinayaklovanshi) [inline app/controllers/wishlist_items_controller.rb:4]: Add extra space

File: app/controllers/wishlists_controller.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+class WishlistsController < ApplicationController
+  before_action :set_wishlist, only: [:show] 
+  def show
```
Comment (vinayaklovanshi) [inline app/controllers/wishlists_controller.rb:4]: Add proper spacing for all the methods

File: app/views/wishlists/show.html.erb [hunk 4]
```diff
@@ -0,0 +1,75 @@
+<style>
```
Comment (vinayaklovanshi) [inline app/views/wishlists/show.html.erb:1]: Add css in css file not in this file.

**Review by vinayaklovanshi — CHANGES_REQUESTED** [review]

File: app/controllers/carts_controller.rb (hunk 1 above)
Comment (rishabhharod) [reply app/controllers/carts_controller.rb:4]: done sir

File: app/controllers/wishlist_items_controller.rb (hunk 2 above)
Comment (rishabhharod) [reply app/controllers/wishlist_items_controller.rb:4]: done

File: app/controllers/wishlists_controller.rb (hunk 3 above)
Comment (rishabhharod) [reply app/controllers/wishlists_controller.rb:4]: spacing added

File: app/views/wishlists/show.html.erb (hunk 4 above)
Comment (rishabhharod) [reply app/views/wishlists/show.html.erb:1]: done sir

**Pull request merged**